lib/core/src/sync/model/data.rs

```diff
@@ -26,6 +26,7 @@ pub(crate) struct ChainSyncData {
     pub(crate) accept_zero_conf: bool,
     pub(crate) created_at: u32,
     pub(crate) description: Option<String>,
+    pub(crate) auto_accepted_fees: bool,
```

Comment:
Just checking (not 100% sure), does this need a `#[serde(default)]` for deserialising?

Reply:
Good point! I think so. Added in [25efa8c](https://github.com/breez/breez-sdk-liquid/pull/690/commits/25efa8c643420232717d1995f457a2ae1ff6c717)